AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -627,6 +627,8 @@ m.blog.naver.com##.btm_banner
 wap.business-standard.com###popup123
 ko.dict.naver.com##.banner_vkorean_wrap
 nv.ua##.selfpromo_new
+m.hindustanttimes.com##.storyDetail > section[style^="clear: both;padding:0px 11px;"]
+||m.hindustantimes.com/res/img/*/app.png
```

Comment:
Click on this element and you'll get a redirect to the Google Play.

Reply:
Well, banner is placed on a website anyway?

Reply:
It's a mobile-app banner, you should move it to the correct section.